hoverxref/extension.py

```diff
@@ -61,10 +61,10 @@ def copy_asset_files(app, exception):
     if exception is None:  # build succeeded
 
         context = {}
-        for attr in app.config.values:
+        for attr, opt in app.config.values.items():
             if attr.startswith('hoverxref_'):
                 # First, add the default values to the context
-                context[attr] = app.config.values[attr][0]
+                context[attr] = getattr(opt, "default", opt[0])
```

Comment:
Won't this still raise a deprecation warning because `opt[0]` is eagerly evaluated? (I have not checked)

Similar for all other replacements.

Reply:
oops, yes!  fixing now...